sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClientTest.java

```diff
@@ -235,7 +235,7 @@ void createsMessageBatchWithSize() {
         int batchSize = 1024;
 
         // Overhead when serializing an event, to figure out what the maximum size we can use for an event payload.
-        int eventOverhead = 46;
+        int eventOverhead = 75;
```

Comment:
This overhead of supporting amqp annotated message

Reply:
Do we use this in our Batch size calculation as well?

Reply:
will create issue to ask SB team.